src/sgame/sg_bot_ai.cpp

```diff
@@ -590,11 +596,17 @@ AINodeStatus_t BotActionDeactivateUpgrade( gentity_t *self, AIGenericNode_t *nod
 	AIActionNode_t *action = ( AIActionNode_t * ) node;
```

Comment:
It seems to me this action should just be deleted, because there is no item in the game that can be meaningfully deactivated. Upgrade activate/deactivation is only used as an implementation detail of grenades and medkits which are activated for 1 frame when you use them.

Reply:
Jetpack?
But as I said earlier, jetpack's usage would not belong to BT anyway, since it directly and only affects the path finding.

Reply:
No. Jetpack activation has nothing to do with this stuff - it's handled in pmove, triggered by the jump button.

Reply:
I'm ok to delete it, but honestly I think #1727 is what should be done instead of random API changes. I kind of doubt it, but maybe @Gireen uses it for example (the ticket I've linked was created to get that kind of infos before doing wasteful efforts).

Reply:
I think the BotActionDeactivateUpgrade changes don't make sense, but the action is useless so it doesn't matter. LGTM